eth/backend.go

```diff
@@ -183,7 +184,11 @@ func New(stack *node.Node, config *ethconfig.Config) (*Quai, error) {
 		config.TxPool.Journal = stack.ResolvePath(config.TxPool.Journal)
 	}
 
-	eth.core, err = core.NewCore(chainDb, &config.Miner, eth.isLocalBlock, &config.TxPool, &config.TxLookupLimit, chainConfig, eth.config.SlicesRunning, eth.config.DomUrl, eth.config.SubUrls, eth.engine, cacheConfig, vmConfig, config.Genesis)
+	if eth.config.Progpow.PowMode == progpow.ModeFake{
```

Comment:
we have both blake3 and progpow now, so might want to check fake mode on both depending on the consensus engine

Reply:
This if statement have been removed on posteriors commits